Automattic/woocommerce-payments
# Use card component from wordpress/components and fix misc related styling issues

Fixes #1327

#### Changes proposed in this Pull Request

This PR switches out use of the woocommerce card component for the card component found in the @wordpress/components package. It updates the transaction details, disputes details, and dispute evidence submission screens. For reference, the WordPress card does not have a drop shadow. It has a 1px boarder, and 2px rounded corners. 

#### Example Before and after

**Transaction details before:**

![screencapture-localhost-wp-admin-admin-php-2021-03-05-15_52_56](https://user-images.githubusercontent.com/4500952/110186798-31aa2780-7dcb-11eb-85c1-d06179c6b401.png)

**Transaction details after:**

![screencapture-localhost-wp-admin-admin-php-2021-03-05-15_49_15](https://user-images.githubusercontent.com/4500952/110186819-4090da00-7dcb-11eb-8530-d0d6ea9c576a.png)

#### Testing instructions

1. Load this branch
2. View the deposit details screen - ensure the deposit details component is wrapped with the WordPress Card component
3. View the transaction details screen - Ensure all cards are using the WordPress Card component
4. View the deposit details and evidence submissions screens - Ensure all cards are using the WordPress Card component

**Follow up issues**

- Revisit the design of the deposit details component on mobile
- [Bump the @wordpress/components version number and base components version number](https://github.com/Automattic/woocommerce-payments/issues/717)

-------------------
